### PR TITLE
added permissions for the usage of CastCrafter/travel_anchors

### PR DESCRIPTION
**Pull request by justliliandev**
# Description

Please write a summary of the changes you have implemented; explaining your design choices if necessary. Please also include a list of things to test if it isn't clear.
Please also disclose whether this is a breaking change.

Added license Information

# Checklist:

- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: doc/license/castcrafter/travel-anchors/license.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+As of today (07.10.21) all current code contributors(this doesn't include translators) noeppi_noeppi, CastCrafter, MelanX, cech12 and Robocraft999 have given permission to use their code for the following purposes:  
+- copy the code as a whole or in part
+- change the license of the copied and modified code to CC0, The Unlicense or any license we might see fit in the future
+
+## The permissions
+![CastCrafter](CastCrafter.png)
+![noeppi_noeppi](noeppi_noeppi.png)
+![MelanX](MelanX.png)
+![cech12](cech12.png)
+![RoboCraft999](RoboCraft999.png)
```
Comment (Rover656) [inline doc/license/castcrafter/travel-anchors/license.md]: broken link :P

Comment (justliliandev) [reply doc/license/castcrafter/travel-anchors/license.md]: which one? they all highlight fine in my ide

Comment (justliliandev) [reply doc/license/castcrafter/travel-anchors/license.md]: all of them, okay

Comment (justliliandev) [reply doc/license/castcrafter/travel-anchors/license.md]: or only robocraft, rich diff doesn't work in PRs for images

**Pull request merged**